package.json

```diff
@@ -106,28 +105,35 @@
   },
   "dependencies": {
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@dotkomonline/design-system": "^0.14.0",
+    "@dotkomonline/react-components": "^2.0.0",
     "@fortawesome/fontawesome-svg-core": "^1.2.14",
     "@fortawesome/free-brands-svg-icons": "^5.8.2",
     "@fortawesome/free-regular-svg-icons": "^5.7.2",
     "@fortawesome/free-solid-svg-icons": "^5.7.1",
     "@fortawesome/react-fontawesome": "^0.1.4",
+    "@mdx-js/loader": "^1.5.7",
     "@sentry/browser": "^5.6.3",
     "@sentry/node": "^5.6.2",
     "@types/classnames": "^2.2.7",
     "@types/express-http-proxy": "^1.5.0",
+    "@types/markdown-to-jsx": "^6.11.0",
     "@types/react-helmet-async": "^1.0.1",
     "@types/react-redux": "^7.1.0",
     "@types/react-select": "^3.0.5",
     "@types/react-stripe-elements": "^1.1.10",
     "@types/redux-thunk": "^2.1.0",
+    "@types/styled-components": "^5.0.1",
     "abortcontroller-polyfill": "^1.2.3",
     "assets-webpack-plugin": "^3.9.7",
     "babel-core": "7.0.0-bridge.0",
     "babel-jest": "^24.1.0",
     "classnames": "^2.2.5",
+    "core-js": "^3.6.4",
     "express-http-proxy": "^1.5.0",
     "focus-visible": "^4.1.5",
     "isomorphic-fetch": "^2.2.1",
+    "markdown-to-jsx": "^6.11.0",
```

Comment:
We already have a markdown lib, why do we need another?

Reply:
There was some issues with importing design system, webpack would refuse to compile without adding markdown-to-jsx to the project. I think this might be solved by adding markdown-to-jsx in designsystems dependencies instead of dev-dependencies.